### DISCUSSION TITLE
Variable height of items

### DISCUSSION THREAD

**Discussion by zoran995** [clauderic/dnd-kit#197] [Q&A] [answered]
I am working on the nested sortable sample, and most of the stuff work great (collisionDetection is not at its best for this case but that is for another discussion).
I am currently facing an issue with the variable height of the items. Draggable is not properly positioning elements on overlay. Everything seems to work correctly on the top level but not in the nested one. Already using `CSS.Translate.toString(transform)` as mentioned in #150. 
When draggable is bigger then item it is dragged over, the overlay is pushed like both items are same size which results in items appear outside the container. When draggable is smaller then the item it is dragged over it appears that overlay is considered to be the same height as item it is dragged over which results in item being pushed to much the bottom.

https://user-images.githubusercontent.com/19208948/114304947-102d1180-9ad6-11eb-87ea-8ce08d3471ef.mp4

**Comment by growthwp** [comment]
@zoran995  I've had no issues with variable heights myself. Can you provide a working codesandbox with the issue?

**Reply by zoran995** [reply]
Quickly created this seems to replicate the issue 
https://codesandbox.io/s/nested-workbench-dnd-kit-5otrf
haven't added `handleDragOver` and `handleDragEnd` code as that part got pretty complex.

**Comment by zoran995** [comment, marked as answer]
Was missing the `strategy={verticalListSortingStrategy}`, after adding it everything works correctly.

**Comment by darren10201** [comment]
@clauderic hey bro this issue still persisting on variable size, I have both verticalListSortingStrategy and CSS.Translate on sortable item.
